ccmlib/scylla_node.py

```diff
@@ -596,6 +597,7 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
         self.is_running()
         if self.scylla_manager and self.scylla_manager.is_agent_available:
             self.start_scylla_manager_agent()
+        self.watch_log_for("S|scylla version.*initialization completed", timeout=5)
```

Comment:
How that related to manager start ?

Reply:
It's not related to the manager, I added it in this line because it's the end of the "start" method.

Reply:
but you already did this inside `_start_scylla`, why do it again ?

Reply:
Also, did you mean `"[Ss]cylla version" etc.?

The regexp you wrote will match either `S` or `scylla version.*initialization completed`.

```
>>> re.match("S|scylla version.*initialization completed", "Scylla version initialization completed")
<re.Match object; span=(0, 1), match='S'>
>>> re.match("S|scylla version.*initialization completed", "scylla version initialization completed")
<re.Match object; span=(0, 39), match='scylla version initialization completed'>
>>> re.match("[Ss]cylla version.*initialization completed", "scylla version initialization completed")
<re.Match object; span=(0, 39), match='scylla version initialization completed'>
```

Reply:
Thanks.
I fixed it.

Reply:
> but you already did this inside `_start_scylla`, why do it again ?

fixed.